md/forms.md

```diff
@@ -24,3 +24,14 @@ The overview consists of three main sections:
 * Timeline: it shows in chronological order information about all the actions that have been performed in the selected case.
 
 You can see how to manage the link between process and and form in the documenation about [live update](live-update.md)
+
+### Application Theme access
+
+If your forms is viewed in an application, you will have access facilities for [the application theme](applications.md).
+
+You can directly access a resource by adding the following link in `index.html`: `<link href="../theme/theme.css" rel="stylesheet" />`
+This link is present by default in the forms generated by the UI Designer. 
+
+::: info It can works thanks to the URL parameter `app`. Since we passe this `app` parameter from the living application layout to the targeted page/forms URL. :::
```

Comment:
do not use `since` if there are no temporal element.
I would have reword the sentence to something like:
Theme is loaded via the `app` URL parameter from the living application layout which the targeted page/form uses inside its URL.

maybe having a look from @NathalieC would be useful